hack/local-up-cluster.sh

```diff
@@ -447,7 +469,7 @@ function start_apiserver {
             fi
         else
             # default to use basic authentication
-            AUTH_ARGS="--username=admin --password=admin"
+            AUTH_ARGS="--username=${KUBE_USER} --password=${KUBE_PASSWORD}"
```

Comment:
I'd rather see this change to reference the generated admin cert/key (going back to something closer to before https://github.com/kubernetes/kubernetes/commit/72e0e91b5edb961cd22e2cdbce5d02300a754402#diff-ed3df710e9af7cd30a185896a60897d9L713)

```
AUTH_ARGS="--client-key=${CERT_DIR}/client-admin.key --client-certificate=${CERT_DIR}/client-admin.crt"
```